### DISCUSSION TITLE
Does enableRigidBodyVTKExport output 2 VTK files? Does it not have a group VTK file？

### DISCUSSION THREAD

**Discussion by duanlinlim** [InteractiveComputerGraphics/SPlisHSPlasH#344] [General]
{
	"Configuration": 
	{
		"pause": true,
		"sim2D": true, 
		"timeStepSize": 0.0004,
		"particleRadius": 0.025,
		"colorMapType": 1,
		"numberOfStepsPerRenderUpdate": 4,
		"density0": 1000, 
		"simulationMethod": 4,
		"gravitation": [0,-981,0], 
		"cflMethod": 1, 
		"cflFactor": 1,
		"cflMaxTimeStepSize": 0.005,
		"maxIterations": 100,
		"maxError": 0.1,
		"maxIterationsV": 100,
		"maxErrorV": 0.1,		
		"stiffness": 50000,
		"exponent": 7,
		"velocityUpdateMethod": 0,
		"enableDivergenceSolver": true,
		"boundaryHandlingMethod": 2,
		"enablePartioExport": true,
		"particleAttributes": "velocity",
		"partioAttributes": "density",
		"enableRigidBodyVTKExport": true,
		"enableRigidBodyExport": true,
		"enableVTKExport": true
		
	},
	"Materials": [
		{
			"id": "Fluid",
			"surfaceTension": 0.2,
			"surfaceTensionMethod": 0,		
			"viscosity": 0.01,
			"viscosityMethod": 1, 
			"vorticityMethod": 1, 
			"vorticity": 0.1, 
			"viscosityOmega": 0.05,
			"inertiaInverse": 0.5,
			"renderMinValue": 0.0,
			"renderMaxValue": 5.0,
			"colorField": "red",
			"colorMapType": 4
		}
	],
	"RigidBodies": [
		{
			"id": 1,
			"geometryFile": "../models/UnitBox.obj",
			"translation": [0,3,0],
			"rotationAxis": [1, 0, 0],
			"rotationAngle": 0,
			"scale": [2, 6, 2.0],
			"color": [0.5, 0.5, 0.5, 1.0], 
			"isDynamic": false,
			"restitution" : 0.6,
			"friction" : 0.0,
			"collisionObjectType": 2,
			"collisionObjectScale": [3.1, 3, 3.1],	
			"testMesh": 1, 
			"isWall": true,
			"invertSDF": true,
			"mapInvert": true, 
			"mapThickness": 0.0,
			"mapResolution": [30,30,30]
		},
		{
			"id": 2,
			"geometryFile": "../models/sphere.obj",
			"isDynamic": false, 
			"density": 1500, 
			"translation": [0.0,5,0],
			"rotationAxis": [1, 0, 0],
			"rotationAngle": 0.0,
			"scale": [0.125, 0.125, 0.125],			
			"restitution" : 0.6,
			"friction" : 0.2,
			"color": [0.3, 0.5, 0.8, 1.0], 
			"collisionObjectType": 1,
			"collisionObjectScale": [0.5, 0.5, 0.5], 
			"testMesh": 1, 
			"isWall": false,
			"mapInvert": false, 
			"mapThickness": 0.0,
			"mapResolution": [25,25,25]
		},
		{
			"id": 3,
			"geometryFile": "../models/sphere.obj",
			"isDynamic": false, 
			"density": 1500, 
			"translation": [0.0,4.5,0],
			"rotationAxis": [1, 0, 0],
			"rotationAngle": 0.0,
			"scale": [0.125, 0.125, 0.125],
			"restitution" : 0.6,
			"friction" : 0.2,
			"color": [0.3, 0.5, 0.8, 1.0], 
			"collisionObjectType": 1,
			"collisionObjectScale": [0.5, 0.5, 0.5], 
			"testMesh": 1, 
			"isWall": false,
			"mapInvert": false, 
			"mapThickness": 0.0,
			"mapResolution": [25,25,25]
		}
	],
	"FluidBlocks": [
		{
			"denseMode": 0,
			"start": [-1, 0.0, -1],
			"end": [1, 6.0, 1],
			"translation": [0.025, 0.025, 0],
			"scale": [1, 1, 1]
		}
	]	The vtk file output from the json file shows that the ball is stationary because enableRigidBody VTKExport only outputs the initial particles, so the ball remains stationary
}

**Comment by janbender** [comment]
The VTK exporter generates one file per static rigid body. If you have dynamics rigid bodies, the exporter will write one file per frame. 

However, it should be quite straightforward to write an own exporter or to adapt the existing one.
